### ISSUE TITLE
JAWS exiting auto forms mode and switching to Browse mode when tabbing to a radiogroup

### ISSUE THREAD

**Issue by anuvenkatesh1** [FreedomScientific/VFO-standards-support#574]
### Summary
Go to https://www.freedomscientific.com/SurfsUp/WebTrack.htm in Chrome
Start JAWS 2020/2021
Tab through the page 
Observe when you tab into the radiogroup JAWS switches to Browse mode ie exits auto forms mode
We have observed this unusual behavior in a few other pages within our application as well (SAP SuccessFactors)

### Expected result
JAWS should remain in auto forms mode

### Actual result
JAWS switched to Browse mode

### Additional Information
https://user-images.githubusercontent.com/32471459/135147007-342cdd3b-4848-4362-ae82-88790edcc299.mp4
We are unable to determine if this is a JAWS bug or something in the code that's causing the switch to Browse mode
As seen in the recording, when we are in the checkbox group, JAWS switched to browse mode automatically which caused it to remain in Browse mode when we tabbed to radiogroup also. However, when we tab past the radiogroup and do a shift+tab to the radiogroup, JAWS enters forms mode. 

What is the cause of this unusual behavior?

#### JAWS version and build number
![jaws_version](https://user-images.githubusercontent.com/32471459/135146876-bb25dcbc-7782-4ba7-b18b-4e998613929d.png)
 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome 92

**Comment by JAWS-test** [comment]
JAWS has always behaved in such a way with radio buttons that the mode is not changed. This means that if JAWS was in Forms Mode before the radio buttons (e.g. coming from an input field), then JAWS remains in Forms Mode. If JAWS was in Browse Mode before the radio buttons (e.g. coming from checkboxes), then JAWS remains in Browse Mode. This is a bit strange, but not a bug; it is known to JAWS users.

**Comment by anuvenkatesh1** [comment]
Thanks much.

**Comment by JAWS-test** [comment]
Although I wouldn't consider this a bug as written, I don't know of any reasonable reason for this behavior, which confuses many new JAWS users. Therefore, it may be considered to abolish this behavior or at least not to offer it in the default configuration of JAWS.
